module-1/pipelines-project/your-code/README.md

```diff
@@ -0,0 +1,85 @@
+# Guided Project: Web Data Pipeline
```

Comment:
Well done in the Readme doc!! You don't need to replicate the functions you have in your main.py unsless it is necesary. It's more important to make a clear description (in human language) of what you want to do, what you need to solve it and what it is your best approach!!